### ISSUE TITLE
Fade in 

### ISSUE THREAD

**Issue by MichaelAmirr** [MichaelAmirr/Mike#1]
<div id="About_us">
        	<div class="titles">
        		<p>About us</p>
        	</div>

        		<p id="golden_rule">
        			
						"The heart and soul of the company is creativity and innovation" Bob Iger
					
				</p>
				
				<div id="img_edwarddebono">
        			<img src="./img/robert_iger 1.png">
        		</div>

				<p id="Aboutus_descreption">
					In Innotivity that's our golden rule, as Bob Iger said; we make our work with very innovative way, creativity also is company's manner so Edward De Bono is company's GOD Father, you can find all his books in our library. We believe both of creativity and innovation lead us to be benchmark in Egypt so in any of our services we must make brain storming sessions to get ideas and then wear 6 thinking hats to be sure that we picked the best one.
				</p>

				<div id="img_edwarddebono">
					<img src="./img/edward-de-bono.png">
				</div>	

					<p id="Ribbon">
						Who we are?	
					</p>

					<p id="Aboutus_descreption">
					
						We are small gang located in Cairo. Our focus is to make marvelous work with competitive prices. Our talented team stood shoulder to shoulder with you to achieve your dreams. Think German that's the way we are thinking, QUALITY is our priority because we believe quality makes an happy customer and that's why we are here. Our passion in using coding allows us to be one of few full services agencies in Egypt. We are always ready to catch your special moments,build your E-commerce empire from scratch with our creative designs & attractive websites using last techniques and our special management to your Facebook page.
					</p>

        	</div>
        	<p>Return to Home page</p>
        	
/*
           Jquery 
*/        	
      
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>

<script src="script.js"></script>

$(function(){  // $(document).ready shorthand
  $('#About_us').fadeIn('slow');
});

$(document).ready(function() {
    
    /* Every time the window is scrolled ... */
    $(window).scroll( function(){
    
        /* Check the location of each desired element */
        $('#golden_rule, #Ribbon, #Aboutus_p , #Aboutus_descreption').each( function(i){
            
            var bottom_of_object = $(this).position().top + $(this).outerHeight();
            var bottom_of_window = $(window).scrollTop() + $(window).height();
            
            /* If the object is completely visible in the window, fade it it */
            if( bottom_of_window > bottom_of_object ){
                
                $(this).animate({'opacity':'1'},2000);
                    
            }
            
        }); 
    
    });
    
});

/*******************************************************************
                                                  CSS
**********************************************************************/
 #About_us{
	text-align: center;
	background: #363F44;
	margin: 0 auto;
	margin-top: -1%;
}
#golden_rule{
	font-style:italic;
	 color:#B4D8F2; 
	 margin-top:2%;
	 font-size: 2.1em;
	font-family: 'Dancing Script', cursive;
	opacity: 0;

}
.titles{
	font-family: 'Bree Serif', serif;
	padding-top: 2%;
	 color: #ffdc19;
	padding-bottom: 1%;
}
.titles p{
	font-size: 4em;
}
#img_edwarddebono img{
	margin-bottom: 2%;
	border-radius: 100%;
	border:solid;
	border-color: #ffdc19;
	border-width: 2px;
	margin-top: 3%;

}
#Aboutus_descreption{
	margin: 0 auto;
	color: #fff;
	font-size: 1.9em;
	width: 70%;
	font-family: 'Karla', sans-serif;
	opacity: 0;
	
}
#Ribbon{
	font-family: 'Bree Serif', serif;
	color: #ffdc19;
	font-size: 2.6em;
	text-transform: uppercase;
	text-align: center;
	margin-top: 1%;
	opacity: 0;
}

**Comment by tonysaad** [comment]
You have to upload all the files to the repo, because now it says it's empty
<img width="513" alt="screen shot 2017-10-16 at 18 01 38" src="https://user-images.githubusercontent.com/1137569/31622228-21572340-b29c-11e7-8bd4-ac38211db250.png">
Now I can't see the problem

**Comment by tonysaad** [comment]
Worked :+1: 
Looking overall: I think you need to show a hidden div #About_us
If that's right, you'd need to add `display: none` in the css ex.
`#About_us{
    display:none;
}`
